lumigator/python/mzai/schemas/lumigator_schemas/workflows.py

```diff
@@ -9,7 +9,7 @@
 class WorkflowCreate(BaseModel):
     name: str
     description: str = ""
-    experiment_id: UUID
+    experiment_id: str
```

Comment:
mlflow created id's (run/experiment) are returned by the mlflow client as strings, not UUIDs. e.g. an experiment id is the string `252773820672265330` and a run id is the string `6c56b97ad4b44c2cafff799bc0e56de0`

Reply:
Ok, so the format does not comply with the uuid representation, and not just a difference in python type hint?

Reply:
Correct. The string `id` i get back from mllfow throws an error when I try to convert it to a UUID, unfortunately.